lucille-core/pom.xml

```diff
@@ -380,6 +380,12 @@
       <artifactId>rssreader</artifactId>
       <version>3.9.3</version>
     </dependency>
+
+    <dependency>
```

Comment:
Should this dependency be commented out with a note to uncomment if OpenSearchAppender is also uncommented in log4j.xml?